src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java

```diff
@@ -232,7 +232,7 @@ public static String getLatestSauceConnectVersion() {
         args = addElement(args, "-P");
         args = addElement(args, String.valueOf(port));
         if (StringUtils.isNotBlank(options)) {
```

Comment:
We should do this here as well:
```suggestion
        if (StringUtils.isNotBlank(options.trim())) {
```

Reply:
👍 Fixed in 739e849